diff --git a/setup.py b/setup.py
index 9e90b67..5c9a2ac 100644
--- a/setup.py
+++ b/setup.py
@@ -64,11 +64,11 @@ setup_kwds = dict(
 
 #setup_kwds['setup_requires'] = ['openalea.deploy']
 
-build_prefix = pj(abspath(dirname(__file__)),"build-scons")
-setup_kwds['scons_scripts'] = ['SConstruct']
-setup_kwds['bin_dirs'] = {'bin': build_prefix + '/bin'}
-setup_kwds['lib_dirs'] = {'lib' : build_prefix+'/lib' }
-setup_kwds['inc_dirs'] = { 'include' : build_prefix+'/include' }
+#build_prefix = pj(abspath(dirname(__file__)),"build-scons")
+#setup_kwds['scons_scripts'] = ['SConstruct']
+#setup_kwds['bin_dirs'] = {'bin': build_prefix + '/bin'}
+#setup_kwds['lib_dirs'] = {'lib' : build_prefix+'/lib' }
+#setup_kwds['inc_dirs'] = { 'include' : build_prefix+'/include' }
 setup_kwds['entry_points']['wralea'] = ['alinea.caribu = alinea.caribu_wralea']
 setup_kwds['package_data'][''] = ['*.can', '*.R', '*.8', '*.opt', '*.light', '*.csv', '*.png']
 
